crates/wiggle/tests/structs.rs

```diff
@@ -299,7 +299,7 @@ impl SumIntAndPtrExercise {
             self.return_loc.ptr as i32,
         );
 
-        assert_eq!(res, Ok(types::Errno::Ok.into()), "sum of int and ptr errno");
+        assert_eq!(res, Ok(types::Errno::Ok as i32), "sum of int and ptr errno");
```

Comment:
@pchickey do you have thoughts on adding back the to/from abi repr traits? I originally removed them because I figured it was good to trim things down to just the in-memory type to avoid too many conversions being available, but it ended up showing up a good amount in tests.

Reply:
I'm fine with either - using `as` in tests is OK with me, or maybe we should do `types::Errno::try_from(res).unwrap()` on the lhs? If you don't want to provide too many `From` impls maybe define a `FromAbi` trait in wiggle that has the same shape, but has docs saying its just for wiggle use?